server.py

```diff
@@ -246,8 +246,30 @@ def get(self, path, feature_id=None):
     self.render(data=template_data, template_path=os.path.join(path + '.html'))
 
 
+class FeaturesAPIHandler(common.JSONHandler):
+
+  def __get_feature_list(self, version=None):
+    feature_list = models.Feature.get_chronological(version=version) # Memcached
+    annotate_first_of_milestones(feature_list, version=version)
+    return feature_list
+
+  def get(self, version=None):
+    if version is None:
+      version = 1
+    else:
+      version = int(version[2:])
```

Comment:
Could you include a comment explaining what the expected value for `version` is going to be? It's not clear why you're unconditionally discarding the first two characters of the string here, so seeing an example would help.